### DISCUSSION TITLE
Problem with ESPresense and iPhone 8

### DISCUSSION THREAD

**Discussion by axelct** [ESPresense/ESPresense#191] [Q&A] [answered]
Hi all,
I think this is a wonderfull project. I've started to use it to track my indoor presence through iPhone 8.
I'm using room assistant companion app to get a fixed UID for my Phone 8 and I've configured sensors (presence and RSSI) with no problem in Home Assistant, but when I'm at home sensor go "in and out" continuosly (as you can see in picture), but i'm alway at home.
<img width="473" alt="Screenshot 2022-01-19 at 09 34 12" src="https://user-images.githubusercontent.com/58512176/150094333-ae0b2da2-385f-43f7-a819-324c84ecbffb.png">
Is there anything I've missing? I've configured espresence qith query and active scan enabled but nothing is changing. 
I think is s iPhone problem but I tought that with room assistant app this didn't happen.
Is anyone can help me?
Thanks in advance.

**Comment by Terastar-Paperclip** [comment, marked as answer]
This is the classic case in our "away detection" reference (#2322) — iPhone screen-off drops the advertising rate, and the rotating identifier makes it look like the device left and a new one arrived. Fix is enrolling the iPhone's IRK plus raising `forget_ms` (default 150s, try 300s). Marking answered off that pin — sorry this one sat quiet for a while.
